content/en/sensitive_data_scanner/_index.md

```diff
@@ -0,0 +1,178 @@
+
+Sensitive Data Scanner is a stream-based, pattern matching service used to identify, tag, and optionally redact or hash sensitive data. Security and compliance teams can implement Sensitive Data Scanner as a new line of defense, helping prevent against sensitive data leaks and limiting non-compliance risks.
+
+To use Sensitive Data Scanner, set up a scanning group to define what data to scan and then set up scanning rules to determine what sensitive information to match within the data.
+
+This document walks you through the following:
+
+- [The permissions required to view and set up Sensitive Data Scanner](#permissions).
+- [Setting up scanning for sensitive data](#set-up-scanning-for-sensitive-data).
+- [Using the out-of-the-box dashboard](#out-of-the-box-dashboard).
+
+{{< img src="sensitive_data_scanner/sds_main_12_01_24.png" alt="The Sensitive Data Scanner page showing six out of the 12 active scanning groups" style="width:90%;">}}
+
+## Permissions
+
+By default, users with the Datadog Admin role have access to view and set up scanning rules. To allow other users access, grant the `data_scanner_read` or `data_scanner_write` permissions under [Compliance][1] to a custom role. See [Access Control][2] for details on how to set up roles and permissions.
+
+{{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}
+
+
+## Set up Sensitive Data Scanner
+
+This section walks you through the following to set up Sensitive Data Scanner:
+
+- [Adding a scanning group](#add-a-scanning-group)
+- [Adding scanning rules](#add-scanning-rules)
+- [Controlling access to events with sensitive data](#control-access-to-events-with-sensitive-data)
+
+You can also [redact sensitive data in tags](#redact-sensitive-data-in-tags).
+
+### Add a scanning group
+
+A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for Logs, APM, RUM, and Events. See the [Log Search Syntax][3] documentation to learn more about query filters.
+
+For Terraform, see the [Datadog sensitive data scanner group][4] resource.
+
+To set up a scanning group:
+
+1. Navigate to [Sensitive Data Scanner][5]. Alternatively, go to Organization Settings > Sensitive Data Scanner.
+1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
+1. Enter a query filter for the data you want to scan. At the top, click **APM Spans** to preview the filtered spans. Click **Logs** to see the filtered logs.
+1. Enter a name and description for the group.
+1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, Logs, APM Spans, RUM Events, and Events).
```

Comment:
```suggestion
1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, logs, APM spans, RUM events, and general events).
```

There may be a better way to say "general events", but it just stuck out to me to say "RUM events and events"

Reply:
Thanks, updated to `Datadog events`